exporter/streams.go

```diff
@@ -52,8 +52,18 @@ func getStreamInfo(c redis.Conn, key string) (*streamInfo, error) {
 	}
 
 	// Extract first and last id from slice
-	stream.FirstEntryId = getStreamEntryId(values, 17)
-	stream.LastEntryId = getStreamEntryId(values, 19)
+	for idx, v := range values {
+		vbytes, ok := v.([]byte)
+		if !ok {
+			continue
+		}
+		if strings.EqualFold(string(vbytes), "first-entry") {
```

Comment:
why not simple `=` equality?

Reply:
I habitually used EqualFold because sometimes there are issues with case conversion. But indeed, you are right, here we can directly use `=`.

Has been modified.